src/core/Akka.Cluster/ClusterEvent.cs

```diff
@@ -69,11 +70,16 @@ public interface IClusterDomainEvent : IDeadLetterSuppression { }
         /// </summary>
         public sealed class CurrentClusterState
         {
+            [JsonProperty]
```

Comment:
Just mark this class with [`INoSerializationVerificationNeeded`](https://getakka.net/api/Akka.Actor.INoSerializationVerificationNeeded.html)

Reply:
Done